src/main/kotlin/dev/restate/sdktesting/infra/RestateContainer.kt

```diff
@@ -102,7 +102,9 @@ class RestateContainer(
                     leaderEnvs +
                     mapOf(
                         "RESTATE_ADVERTISED_ADDRESS" to
-                            "http://$RESTATE_RUNTIME:$RUNTIME_NODE_PORT"),
+                            "http://$RESTATE_RUNTIME:$RUNTIME_NODE_PORT",
+                        "DO_NOT_TRACK" to "true",
```

Comment:
In the init block below, not here :) this is used only in the multi container setup